index.html

```diff
@@ -76,6 +76,20 @@ <h2>CapturedMouseEvent interface</h2>
         more scrutiny in which information may safely be exposed. At present, only information
         already available to the capturing application is exposed - the mouse coordinates.
       </p>
+      <p>
+        For any <a data-cite="screen-capture/#dfn-capture-session">capture-session</a>
+        that has an associated {{CaptureController}}, events with type
+        <dfn>capturedmousechange</dfn>, which do not bubble, which are not
+        cancelable and which use the {{CapturedMouseEvent}} interface, MUST be
+        regularly dispatched on this {{CaptureController}} to indicate any
+        move of the captured cursor within the [=display surface=] associated
+        to that capture-session, or whether the captured cursor entered/exited
+        that surface.
+      </p>
+      <p>Events MUST NOT be fired more than 30 times per second.</p>
+      <p>
+        Events MUST NOT be fired after the last clone of the original video track is stopped.
```

Comment:
I know this is moved text too, but could you change to a formulation that more clearly explains that the original video itself is also considered, not just its clones?